O/OptimizationBase/Compat.toml

```
@@ -1,4 +1,4 @@
-["0-0.0.5"]
+[0]
```

Why not
```suggestion
["0-1.0.0"]
```
and remove the duplicate section below?  I'm surprised RegistryCI doesn't compress it like that already.

---

Because I was not sure since also currently on master the ADTypes compat for 1.0.0 is not combined with the other versions.